python/scripts/validate_datasets.py

```
@@ -0,0 +1,492 @@
+            output = json.loads(result.stdout.strip().split('\n')[-1])
+            return ValidationResult(
+                dataset_name=dataset["name"],
+                dataset_url=dataset["url"],
+                spatialdata_version=version,
+                success=output.get("success", False),
+                error_type=output.get("error_type"),
+                error_message=output.get("error_message"),
+                elements=output.get("elements"),
+                coordinate_systems=output.get("coordinate_systems"),
+            )
+        except (json.JSONDecodeError, IndexError) as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)